src/org/exist/client/CommandlineOptions.java

```diff
@@ -21,157 +21,296 @@
  */
 package org.exist.client;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.io.File;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.util.*;
 
-import org.apache.avalon.excalibur.cli.CLOptionDescriptor;
 import org.exist.xmldb.XmldbURI;
+import org.exist.xquery.util.URIUtils;
+import se.softhouse.jargo.*;
+
+import static org.exist.util.ArgumentUtil.*;
+import static se.softhouse.jargo.Arguments.*;
 
 /**
+ * Command Line Options for the {@link InteractiveClient}
  *
- * @author wessels
+ * @author Adam Retter <adam.retter@googlemail.com>
```

Comment:
@adamretter That is not me but copyright law. The work of others must be respected even if it's just a few words ;-)